### ISSUE TITLE
Injecting files with a query string does not work

### ISSUE THREAD

**Issue by ludohenin** [klei/gulp-inject#107]
I have a custom transform function which adds a query string version like `my-script.js?v=0.0.1`.
This does not work, the file is not injected.

**Comment by joakimbeng** [comment]
Can you please provide a full example of how you're using `gulp-inject` to do this, to make it easier to help you.

**Comment by ludohenin** [comment]
Transform function:
```javascript
function (filepath) {
  arguments[0] = filepath + '?v=' version;
  return inject.transform.apply(inject.transform, arguments);
}
```

**Comment by joakimbeng** [comment]
@ludohenin see the test I just commited for a proof that it does actually work.

**Comment by ludohenin** [comment]
@joakimbeng I just ran the test and I confirm it doesn't work. Sorry but I should have been more specific, if you change `version = '1.0.0'` (in test and expectation) then the test fails, the file doesn't get injected.

**Comment by joakimbeng** [comment]
Fixed in `1.3.1`! :)

**Comment by ludohenin** [comment]
Superb, thanks.
